### PR TITLE
chore(github): Update issue template to have several choices

### LINKED ISSUE #13648: Update github templates when submitting new issues
### Description
Today, we ask OS and version in template, while we should ask as well the type of the underlying platform (minikube, minishift, openshift, k8s, microK8s, etc.) and if it has been deployed by helm, minishift-addon or che-operator.

It would be nice to have `chectl` allowing to help when providing logs/relevant diagnostic information

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Provide a list of choices when creating issues

Enhance the templates to ask more questions when reporting a bug
image

### What issues does this PR fix or reference?
Related to End Game https://github.com/eclipse/che/issues/13637

Fix https://github.com/eclipse/che/issues/13648

Change-Id: I3586957c6687dbdf4b528ccf62bd07e82b2887f1
Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### CODE REVIEW

**Comment by slemeur** [comment]
I love it !

I'm wondering if we need the usability section as this might be related to enhancements?

Order, I would put:
- Question
- Bug report
- Enhancement
- Usability
- Documentation
- Task


But that's great. Thanks @benoitf !

**Review by slemeur — APPROVED** [review]

**Comment by benoitf** [comment]
@slemeur it's because we have `kind/usability` label
It seems @tsmaeder introduced that label 
Maybe you need to see if `usability` can be merged into `enhancement`

File: .github/ISSUE_TEMPLATE/p_doc.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Documentation 📘
+about: Suggest a documentation item for Eclipse Che
+title: ''
+labels: 'kind/docs'
+assignees: ''
```
Comment (benoitf) [inline .github/ISSUE_TEMPLATE/p_doc.md]: @rkratky we can make you as default assignee as well there

Also we can add the template that you would want for documentation items.

**Comment by l0rd** [comment]
Very nice @benoitf . +1 to remove usability. I would even make it simpler and remove:

- documentation: for me it's an area or component, not a kind. For instance it makes perfectly sense to have `kind/enhancement` with `area/docs`
- task: I always struggle with the difference between a task and an enhancement 



 (kind/enhancement + area/docs or component/che-docs should be better)

**Review by l0rd — APPROVED** [review]

**Comment by benoitf** [comment]
ok so consensus for now is to remove usability (and then remove the label I think ?)

I agree with you that `kind/docs` is more `area/docs` now that docs issues are also there 

About task, in description (that is in the label) is that it's more technical-debt, something that you should do but it may not be an enhancement at all.
Usage of `kind/task` is quite widely spread with almost 200 opened issues with that label https://github.com/eclipse/che/issues?q=is%3Aopen+is%3Aissue+label%3Akind%2Ftask
If most of people are thinking that task can be merged into enhancement we could reduce the numbers of `kind/` item as well

I will not do changes on the PR until beginning of next week to let people comment and grab other feedback.

**Comment by sunix** [comment]
> I will not do changes on the PR until beginning of next week to let people comment and grab other feedback.

Ok, my feedback is that to make some decision like what we take for the release, we take some criteria like: 
- "We do not add more new features for GA" --> `enhancement` (but should be `new feature` ?)
- "We have to update the README to include a contribution section" --> `task` or `doc`
- "We have to update the jenkins job to include the ability to build PR branches" --> `task`
- "this flow is ok but it would be nicer to have the button on the top" --> `usability` .. not a new feature, not really a bug, UX.

Finally is `enhancement` that would englobe several kinds of issue a good thing that will help in taking decisions on what we move forward ? I don't think so.

**Review by rhopp — APPROVED** [review]

**Comment by benoitf** [comment]
@sunix I think you can address some of your remarks with naming convention in your commits
https://www.conventionalcommits.org/

**Comment by benoitf** [comment]
ci-build

**Comment by benoitf** [comment]
@slemeur @l0rd PR updated

**Comment by slemeur** [comment]
LGTM

**Review by l0rd — APPROVED** [review]

**Comment by sunix** [comment]
> @sunix I think you can address some of your remarks with naming convention in your commits
> https://www.conventionalcommits.org/

yes but before coding (when filling the issue) and while planning (choosing what issue to do next), how can I address that ?

**Comment by benoitf** [comment]
you can still give issue/PR title with this convention (look at this PR title)

**Pull request merged**